packages/xo-vmdk-to-vhd/src/parseVhdToBlocks.js

```diff
@@ -0,0 +1,41 @@
+import assert from 'assert'
+import { parseVhdStream } from 'vhd-lib'
+
+async function next(iterator, type, skipableType) {
+  let item
+  do {
+    const cursor = await iterator.next()
+    assert.strictEqual(cursor.done, false, 'iterator is done')
+
+    item = cursor.value
+  } while (item.type === skipableType)
+  assert.strictEqual(item.type, type)
+  return item
+}
+
+async function* onlyBlocks(iterable, blockSize) {
+  for await (const item of iterable) {
+    if (item.type === 'block') {
+      // transform to blocks expected by generateVmdkData
+      // rename the property and remove the block bitmap
+      const { id, data } = item
+      yield { lba: id * blockSize, block: data.slice(data.length - blockSize) }
```

Comment:
`data` is already the block's content.

```suggestion
      yield { lba: id * blockSize, block: data }
```